### ISSUE TITLE
I found 2 bugs

### ISSUE THREAD

**Issue by Neprogrammist222** [irixxxx/picodrive#137]
I found 2 bugs in the latest picodrive emulator. In the game another world, the upper part of the screen flickers, when you connect the set-top box to the TV, the TV screen also flickers through the TV output. When overclocking the processor (settings -advanced settings - overclocking), the sound is interrupted in the game road rash 3, there is no sound at all. there is no such error in older versions. I have an rs-97 v3.0 with ips screen, retrofw 2.3 with the latest kernel

**Comment by irixxxx** [comment]
Try setting the h/v scaling to hardware. Is this still the case?

**Comment by Neprogrammist222** [comment]
Yes, the problem remains. the top of the screen is flickering.

**Comment by irixxxx** [comment]
Could you explain where in the game you are seeing this? I'm not able to replicate it by running the intro, then starting the game to the 1st scene.

**Comment by Neprogrammist222** [comment]
now I understood what the reason for the error was, another world was playing at 50 hz, and the region was running at 60 hz in the emulator. because of this, there was a bug with the screen. as soon as I changed the region to eu, the flickering disappeared. Everything is fine now. and in road rash too. It looks like the problem has been solved. I uploaded the video to Telegram
https://t.me/+nlCccsVs60JmMWRi
